tests/config/test.yaml

```diff
@@ -2,15 +2,13 @@ general:
   description: example configuration
   name: example
 input_data:
-  bucket: solar-pv-nowcasting-data
-  npw_base_path: tests/data/nwp_data/test.zarr
-  satelite_filename: tests/data/sat_data.zarr
+  nwp_zarr_path: tests/data/nwp_data/test.zarr
+  satellite_zarr_path: tests/data/sat_data.zarr
   solar_pv_data_filename: tests/data/pv_data/test.nc
   solar_pv_metadata_filename: tests/data/pv_metadata/UK_PV_metadata.csv
-  solar_pv_path: tests/data/pv_data
-  gsp_filename: tests/data/gsp/test.zarr
+  gsp_zarr_path: tests/data/gsp/test.zarr
 output_data:
-  filepath: solar-pv-nowcasting-data/prepared_ML_training_data/v5/
+  filepath: NA
```

Comment:
I like this, would it be even more verbose syaing filepath: not-setting-this-as-its-for-a-test

Reply:
Good idea!  Done!